proxy/envoy/config.go

```diff
@@ -170,7 +173,7 @@ func buildIngressListeners(instances []*model.ServiceInstance) ([]Listener, []Cl
 				Config: NetworkFilterConfig{
 					CodecType:  "auto",
 					StatPrefix: "ingress_http",
-					AccessLog:  []AccessLog{{Path: "/dev/stdout"}},
+					AccessLog:  []AccessLog{{Path: "/dev/null"}},
```

Comment:
I would rather keep it to either stdout or some file configurable by user. The monitoring reports that go the mixer are aggregate metrics. Until we have zipkin support for envoy, losing detailed access log seems not a good idea.